zapcore/console_encoder.go

```diff
@@ -56,6 +56,10 @@ type consoleEncoder struct {
 // encoder configuration, it will omit any element whose key is set to the empty
 // string.
 func NewConsoleEncoder(cfg EncoderConfig) Encoder {
+	if len(cfg.ConsoleSeparator) <= 0 {
```

Comment:
minor: len can never be `< 0`.

```suggestion
	if len(cfg.ConsoleSeparator) == 0 {
```